source/sonata_config.rst

```diff
@@ -10,10 +10,29 @@ version
 
 *Optional*.
 
-An integer defining the version of the SONATA specification.
+An number defining the version of the SONATA specification.
```

Comment:
An -> a

Reply:
fixed

Reply:
Do we have fractional versioning?

Reply:
by accident:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/13150119/200512332-ad9de1ce-a5b8-4457-adc3-ed33a03cd565.png">

Reply:
:see_no_evil:

Reply:
Wait a sec, that spec part is contradicting itself?

Reply:
how?

Reply:
"An integer defining the version" "This version is "2.3""

Reply:
Last time I checked, 2.3 wasn't exactly an integer.

Reply:
Yeah, I think it's partly b/c it's not obvious how `|CurrentSonataConfigVersion|.` was resolved.

I could probably make this more `correct` by saying a `semver style number` instead of just `A number`?

Reply:
> Last time I checked, 2.3 wasn't exactly an integer.
Ah yeah, but isn't that exactly what Mike fixed above?

Reply:
Ah, I see now, was too focused on the screenshot (also thought this came from upstream, somehow)! :facepalm: 

`semver style number` seems pretty good.